src/content/guides/language/sound-problems.md

```diff
@@ -494,6 +494,35 @@ var maximumOrNull =
 
 <hr>
 
+### Conflicting Superinterfaces
+
+A class which `implements` more than one superinterface must be able to
+implement valid overrides for every member.
+Each member with a given name requires compatible signatures across the
+superinterfaces.
```

Comment:
**suggestion:** May want to add something to clarify here.
```suggestion
A class which `implements` more than one superinterface, or member,
must be able to implement valid overrides for every member.
Each member with a given name requires compatible signatures across the
superinterfaces.
```

If this is done elsewhere, you don't need to do it here.

Reply:
I don't think this is correct - I'll tweak the wording for clarity. I'll change to.

```suggestion
A class which `implements` more than one superinterface must be able to
implement valid overrides for every member of every superinterface.
Each member with a given name requires compatible signatures across the
superinterfaces.
```